flake.nix

```diff
@@ -1,77 +1,60 @@
 {
   inputs = {
-
     flake-utils.url = "github:numtide/flake-utils";
-
     naersk.url = "github:nix-community/naersk";
-
     nixpkgs.url = "github:NixOS/nixpkgs/nixpkgs-unstable";
-
     treefmt-nix.url = "github:numtide/treefmt-nix";
-
     rust-overlay.url = "github:oxalica/rust-overlay";
-
   };
 
   outputs = {
-      self
-    , flake-utils
-    , naersk
-    , nixpkgs
-    , treefmt-nix
-    , rust-overlay
```

Comment:
I actually prefer this, it's the same style that https://github.com/nixos/nixpkgs uses.

Reply:
There seems to already have been a lot of discussion on the topic: https://github.com/kamadorueda/alejandra/issues/95

I don't care that much at the end of the day. I use alejandra because I found it's formatting pretty helpful but unintrusive working with it. (Style discussions are some of the most annoying things devs - let's be honest - fight about imo)

Judging by context I assume you don't insist on using something else?!